### ISSUE TITLE
Rules for Text Resizing 

### ISSUE THREAD

**Issue by JunBadger** [redbadger/pride-london-app#128]
In order to keep design from breaking and maintain hierarchy the following rules should be applied:

- H2: max-size = 22px (up from 18px)
- H3: max-size = 20px (up from 16px)
- H4: max-size = 18px (up from 16px)
- Para: stays at 16px (no resizing)
- Small text: max-size = 16px (up from 14px)

Line-heights should stay proportional. Is this currently measured in ems?

**Comment by cellyn-tan** [comment]
@daralthus ^ for your reference

**Comment by daralthus** [comment]
We can do something like this in `./components/Text.js` to reverse the effect of accessibilty font scaling after a certain ratio

```
h2: {
    fontFamily: "Poppins-SemiBold",
    fontSize:
      PixelRatio.getFontScale() < 1.222 ? 18 : 22 / PixelRatio.getFontScale(),
    lineHeight:
      PixelRatio.getFontScale() < 1.222 ? 24 : 26 / PixelRatio.getFontScale(),
    includeFontPadding: false
  },
```
